### PR TITLE
Implement data generation to CSV text file

### PR DESCRIPTION
**Pull request by thakichow**
Ticket: https://trello.com/c/9KkZZ6qR/10-sm-create-algorithm-to-generate-dummy-data
@allisonc07 Please review data generation code when you have a moment!

### CODE REVIEW

File: database/generate-data.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+  })
+
+};
+
+const getRandomIndex = (num) => {
+  // i: number
+  // o: number, random between 0 and num
+
+  function getRandomIntInclusive(min, max) {
```
Comment (allisonc07) [inline database/generate-data.js]: Try refactoring to ES6 style for consistency

Comment (allisonc07) [reply database/generate-data.js]: Also, does this need to be an inner function? Maybe you can refactor so there is no inner function

File: database/generate-data.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  return first.toUpperCase() + remainder;
+}
+
+const generateExperience = () => {
+  // i: nothing
+  // o: string
+
+  let result = ``;
+
+  for (let i = 0; i <= 100; i++) {
```
Comment (allisonc07) [inline database/generate-data.js]: I think the Airbnb style guide doesn't recommend using unary operators, they suggest using (i += 1) instead

**Comment by thakichow** [comment]
Thanks for the suggestions! Please see updated version of file.

**Pull request merged**